nym-data-observatory/docker-compose.yml

```diff
@@ -2,12 +2,22 @@ services:
   postgres:
     image: postgres:13
     container_name: nym-data-observatory-pg
-    env_file:
-      - .env
+    environment:
+      POSTGRES_PASSWORD: password
     ports:
       - "5432:5432"
     volumes:
       - pgdata:/var/lib/postgresql/data
 
+  data-observatory:
```

Comment:
can you add postgres as a dependency of data-observatory? it needs postgres to run